app/models/facility.rb

```diff
@@ -246,6 +246,7 @@ def locale
       "default" => "am-ET",
       "dire dawa" => "am-ET",
       "oromia" => "om-ET",
+      "sidama" => "sid-ET",
```

Comment:
Not directly related to the PR. We added a new language and I'm adding it to our locales map.